public/javascripts/SVLabel/spec/SVLabel/modal/ModalMissionCompleteMap-spec.js

```diff
@@ -0,0 +1,332 @@
+    } 
+
+    TaskMock.prototype.getStreetEdgeId = function () {
+        return this.properties.streetEdgeId;
+    };
+
+    TaskMock.prototype.getFeature = function () {
+        return this.properties.feature;
+    };
+
+    TaskMock.prototype.getGeoJSON = function () {
+        var geoJSON = {
+            "type":"FeatureCollection",
+            "features": [this.properties.feature]
+        };
+        return geoJSON;
+    };
+
+    beforeEach(function () {
+        $uiModalMissionCompleteFixture = $('<div id="modal-mission-complete-map"></div>');
+
+        uiModalMissionComplete = {};
+        uiModalMissionComplete.map = $uiModalMissionCompleteFixture;
+
+        this.uiModalMissionComplete = uiModalMissionComplete;
+        
+        map = new ModalMissionCompleteMap(uiModalMissionComplete);
+        
+        mission = new MissionMock();
+        mission.properties.distanceMi = 0.7575;
+        mission.properties.distance = 1219.2;
+        mission.properties.distanceFt = 4000;
+        mission.properties.coverage = 0.07575;
+        mission.properties.auditDistanceFt = 2000;
+        mission.properties.auditDistanceMi = 0.3788;
+        mission.properties.auditDistance = 609;
+        mission.properties.label = "distance-mission";
+        neighborhood = new NeighborhoodMock();
+        neighborhood.properties.name = "Test Neighborhood";
+        neighborhood.properties.geoJSON = {
+               "type":"Feature",
+               "properties":{
+
+               },
+               "geometry":{
+                  "type":"Polygon",
+                  "coordinates":[
+                     [
+                        [
+                           -76.9238,
+                           38.9035
+                        ],
+                        [
+                           -76.9239,
+                           38.9032
+                        ],
+                        [
+                           -76.9240,
+                           38.9028
+                        ]
+                     ]
+                  ]
+               }
+            };
+    });
+
+    //remove the html map container after each run
+    afterEach( function () {
+      $('#modal-mission-complete-map').remove();
+    });
+
+    describe("ModalMissionCompleteMap object declaration", function () {
+      it("should initialize attributes", function () {
+        expect(map).not.toBe(null);
+        expect(map._southWest).not.toBe(null);
+        expect(map._northEast).not.toBe(null);
+        expect(map._bound).not.toBe(null);
+        expect(map._map).not.toBe(null);
+        expect(map._overlayPolygon).toBe(null);
+        expect(map._overlayPolygonLayer).toBe(null);
+        expect(map._ui).not.toBe(null);
+      });
+    });
+
+    describe("linestringToPoint method", function(){
+        it("should convert LineString to Point GeoJSON", function() {
+          // GeoJSON FeatureCollection of LineStrings
+          var c = {"type":"FeatureCollection",
```

Comment:
Is this converting the `FeatureCollection` of `LineString`s to another `FeatureCollection` of `Point`s? If so, this is good! But can also check change the title of the spec to `"should convert a FeatureCollection of LineStrings to a FeatureCollection of Points"`? And write another spec that checks the top level GeoJSON element's type remains the `FeatureCollection`?